mindmeld/components/_config.py

```diff
@@ -102,97 +103,91 @@
 
 DEFAULT_ENTITY_RESOLVER_CONFIG = {"model_type": "text_relevance"}
 
+DEFAULT_QUESTION_ANSWERER_CONFIG = {"model_type": "keyword"}
+
 ENGLISH_LANGUAGE_CODE = "en"
 ENGLISH_US_LOCALE = "en_US"
 DEFAULT_LANGUAGE_CONFIG = {
     "language": ENGLISH_LANGUAGE_CODE,
     "locale": ENGLISH_US_LOCALE,
 }
 
-DOC_TYPE = "document"
 
 # ElasticSearch mapping to define text analysis settings for text fields.
 # It defines specific index configuration for synonym indices. The common index configuration
 # is in default index template.
 DEFAULT_ES_SYNONYM_MAPPING = {
```

Comment:
Is it possible to prevent deprecation by having one such mapping for <ES 7.0 and one for 7.0?

Reply:
Yeah good point, will see how we can make this backwards compatible.

Reply:
Note: the code has been updated so it is fully backwards compatible.